schedule/virt_autotest/kubevirt-tests-agent.yaml

```diff
@@ -0,0 +1,29 @@
+name:           kubevirt-tests-agent
+description:    >
+    Maintainer: nan.zhang@suse.com
+    Kubevirt agent node installation and test modules
+vars:
+    SCC_ADDONS: legacy,containers
+schedule:
+    - boot/boot_from_pxe
+    - installation/welcome
+    - installation/accept_license
+    - installation/scc_registration
+    - installation/addon_products_sle
+    - installation/system_role
+    - installation/partitioning
+    - installation/partitioning_firstdisk
+    - installation/partitioning_finish
+    - installation/installer_timezone
+    - installation/user_settings
+    - installation/user_settings_root
+    - installation/resolve_dependency_issues
+    - installation/installation_overview
+    - installation/disable_grub_timeout
+    - installation/disable_grub_graphics
+    - installation/start_install
+    - installation/await_install
+    - installation/logs_from_installation_system
+    - installation/reboot_after_installation
+    - virt_autotest/login_console
```

Comment:
Suggest to add install_package and update_package, which will ensure system and virt packages are up-to-date to MU channels.

Reply:
Final decision is to remove these two modules.